perennialAutodidact/memory-snap
# 43 setup sockets

﻿## Description of changes 
- Setup absolute imports in the backend package (probably could have been a separate ticket, my bad)
- Installed and setup socket.io and socket.io-client
- Setup helper function to create a test socket with the same event handlers as the actual app socket
- Moved photo api logic into a separate function

## Test steps
There aren't really any test steps.

All the socket stuff was added in order to get the single test in `packages/backend/controllers/socketio/index.test.js` to work.

The absolute imports could be tested if you go into `backend/app.js`, type `import {} from '__'` and verify that intellisense suggests the `__mocks__` and `__test__` directories.
